docs/basic-guide/data-type.mdx

```diff
@@ -724,11 +725,11 @@ SELECT
 
 ```
 
-## 元组中的数据类型
+## Data types in tuples
 
-在动态创建元组时，byconity 会自动为元组的每一个参数赋予最小可表达的类型。如果参数为 [NULL](https://clickhouse.com/docs/zh/sql-reference/data-types/tuple#null-literal)，那这个元组对应元素是 [可为空](https://clickhouse.com/docs/zh/sql-reference/data-types/nullable)。
+When dynamically creating tuples, ByConity will automatically assign the smallest expressible type to each parameter of the tuple. If the parameter is [NULL](https://clickhouse.com/docs/zh/sql-reference/data-types/tuple#null-literal), then the corresponding element of this tuple is [nullable](https:/ /clickhouse.com/docs/en/sql-reference/data-types/nullable).
```

Comment:
The first link should use the English one, and the second link has a space need to be removed.